src/Sylius/Bundle/ApiBundle/Resources/config/services/api_payment_methods.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service id="Sylius\Bundle\ApiBundle\Provider\PaymentConfigurationProvider">
+            <argument type="tagged_iterator" tag="api.payment" />
```

Comment:
Can you explain why did you add here this tag? and how is it work?

Reply:
Alright, once we create Class with interface ApiPaymentMethodInterface we can tag this service in our plugin or payment extension and it will load it automatically to PaymentConfigurationProvider as it takes all services tagged with api.payment. When this interface is in core there is another thing to do and auto tag every single class which implements ApiPaymentMethodInterface, this way there wont be need for tagging every service, all you need to do is provide class with given interface and our provider will take care of it ;) (generic solution i guess xD)

Reply:
but for now tagging every service works enough, ill remember about refactor before we implement any other payments to api